src/main/java/org/apache/maven/plugins/javadoc/AbstractJavadocMojo.java

```diff
@@ -6144,6 +6171,23 @@ private boolean containsWarnings( String output )
         }
     }
 
+    /**
+     * Determines whether the specified string is informational output of the Javadoc tool.
+     *
+     * @param str string to check
+     * @return true if informational output, false if not or cannot be determined
+     */
+    private boolean isInfoOutput( String str )
+    {
+        return str == null
```

Comment:
Last question. Why do you consider a null string informational? Do you expect a line to be `null` and not `""`?

Reply:
A null string should not go to the non-informational output therefore it is considered informational. The null check is also to ensure the following statements will never cause NPE.
You are right, I should include empty and blank strings.